public/app/features/dashboard-scene/scene/layout-default/DefaultGridLayoutManager.tsx

```diff
@@ -349,7 +374,48 @@ export class DefaultGridLayoutManager
     });
   }
 
+  public renderEditor() {
+    return <DefaultGridLayoutEditor layoutManager={this} />;
+  }
+
   public static Component = ({ model }: SceneComponentProps<DefaultGridLayoutManager>) => {
-    return <model.state.grid.Component model={model.state.grid} />;
+    if (!config.featureToggles.dashboardNewLayouts) {
+      return <model.state.grid.Component model={model.state.grid} />;
+    }
+
+    return (
+      <LayoutEditChrome layoutManager={model}>
+        <model.state.grid.Component model={model.state.grid} />
+      </LayoutEditChrome>
```

Comment:
I think it might in that having in the canvas is the best and provides the easiest and most intuitive UX (esp to easily support nested layouts like Tabs), a tree view on the side might be another more complex solution but worried about the lack of space for dashboard view and a side pane on laptops with limited screen space